sssom/util.py

```diff
@@ -760,9 +761,9 @@ def deal_with_negation(df: pd.DataFrame) -> pd.DataFrame:
 
     # GroupBy and SELECT ONLY maximum confidence
     max_confidence_df: pd.DataFrame
-    max_confidence_df = combined_normalized_subset.groupby(
-        KEY_FEATURES, as_index=False
-    )[CONFIDENCE].max()
+    max_confidence_df = combined_normalized_subset.groupby(TRIPLES_IDS, as_index=False)[
```

Comment:
If this `combined_normalized_subset` has no negative mappings in this context, this is ok, otherwise why not use `KEY_FEATURES`?